src/cmdc_tools/datasets/owid/__init__.py

```diff
@@ -35,7 +35,7 @@ def get(self):
         url = "https://covid.ourworldindata.org/data/owid-covid-data.csv"
         return (
             pd.read_csv(url, parse_dates=["date"])
-            .rename(columns={"date": "dt"})
+            .rename(columns={"date": "dt", "cardiovasc_death_rate": "cvd_death_rate"})

```

Comment:
OWID renamed their cardiovascular death rate variable which was causing tests to break. See [this commit](https://github.com/owid/covid-19-data/commit/fa29d46b2a1fc371b824a55dcc230e915ed2f8a8)